Projects/Core/DesignSystem/Sources/CheckBox/DotoriCheckBox.swift

```diff
@@ -0,0 +1,126 @@
+import UIKit
+
+public final class DotoriCheckBox: UIControl {
+
+    private let checkedView: UIImageView = {
+        let view = UIImageView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.isHidden = true
```

Comment:
```suggestion
        view.isHidden = !isChecked
```
생성자에서 isChecked를 받으니 위처럼 해야할거같아요.